static/js/activity.js

```diff
@@ -103,6 +104,10 @@ export function redraw_user(user_id) {
         return;
     }
 
+    if (muting.is_user_muted(user_id)) {
+        return;
+    }
```

Comment:
Would it be cleaner to put this check in `buddy_list.insert_or_move`?  That seems better given that the main enforcement of this behavior is in the `buddy_list` redraw logic.

It's a little awkward because for some reason `buddy_data` doesn't call its key `user_id`, but I'm not sure why we do that.

@showell thoughts?